Integrations/integration-Thinkst_Canary.yml

```diff
@@ -0,0 +1,558 @@
+description: By presenting itself as an apparently benign and legitiamte service(s), the canary draws the attention of
+  unwanted activity. When someone trips one of the Canary's triggers, an alert is sent to notify the responsible
+  parties so that action can be taken before valubale systems in your network are compromised.
+configuration:
+- display: Fetch incidents
+  name: isFetch
+  defaultvalue: ""
+  type: 8
+  required: false
+- display: Incident type
+  name: incidentType
+  defaultvalue: ""
+  type: 13
+  required: false
+- display: Canary Server URL
+  name: server
+  defaultvalue: ""
+  type: 0
+  required: true
+- display: API auth token
+  name: auth_token
+  defaultvalue: ""
+  type: 4
+  required: true
+- display: Trust any certificate (unsecure)
+  name: insecure
+  defaultvalue: ""
+  type: 8
+  required: false
+- display: Use system proxy settings
+  name: proxy
+  defaultvalue: ""
+  type: 8
+  required: false
+script:
+  script: |-
+    ''' IMPORTS '''
+    import requests
+    import yaml
+    import time
+
+    # disable insecure warnings
+    requests.packages.urllib3.disable_warnings()
+
+    # remove proxy if not set to true in params
+    if not demisto.params()['proxy']:
+        del os.environ['HTTP_PROXY']
+        del os.environ['HTTPS_PROXY']
+        del os.environ['http_proxy']
+        del os.environ['https_proxy']
+
+    ''' GLOBALS'''
+
+    SERVER = demisto.params().get('server').rstrip('/') + '/api/v1/'
+    AUTH_TOKEN = demisto.params().get('auth_token')
+    UNSECURED = demisto.params().get('insecure')
+    FETCH_DELTA = 24
```

Comment:
this should be configurable, please add parameter for it

Reply:
This is by design, this type of product will not normally have more than 10 max incidents per day, so no reason to make it configurable. We want to make the instance configuration as user-friendly as possible.

Reply:
not sure what's the point here..
if the user wants to fetch alerts from the last day / week / month, we should let him/her
the default can be 24 hours, and then it does not even impact the ux
this became our convention after understanding that we thought to be the "normal" is not exactly that
please do make it configurable